DataConnectors/CyclancePROTECT/Connector_Syslog_CylancePROTECT.json

```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "BlackberryCylancePROTECT",
+    "title": "Blackberry CylancePROTECT", 
+    "publisher": "Blackberry", 
+    "descriptionMarkdown": "The [Blackberry CylancePROTECT](https://www.blackberry.com/us/en/products/blackberry-protect) connector allows you to easily connect your CylancePROTECT logs with Azure Sentinel This gives you more insight into your organization's network and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **CylancePROTECT** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-cylanceprotect-parser)",
+    "graphQueries": [ 
+     { 
+         "metricName": "Total data received", 
+         "legend": "CylancePROTECT", 
+         "baseQuery": "CylancePROTECT" 
+     } 
+    ], 
+          "sampleQueries": [ 
+           { 
+               "description" : "Top 10 Event Types", 
+               "query": "CylancePROTECT \n | summarize count() by EventName \n| top 10 by count_"    
+           },
+            {
+                "description" : "Top 10 Triggered Policies",
+                "query": "CylancePROTECT \n | where EventType == \"Threat\" \n | summarize count() by PolicyName \n | top 10 by count_\n| top 10 by count_"
+            }
+        ], 
+            "dataTypes": [ 
+            { 
+               "name": "Syslog (CylancePROTECT)", 
+               "lastDataReceivedQuery": "CylancePROTECT \n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)" 
+            } 
+        ], 
+             "connectivityCriterias": [ 
+                { 
+                    "type": "IsConnectedQuery", 
+                    "value": [ 
+                    "CylancePROTECT \n      | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)" 
+             ] 
+                } 
+            ], 
+            "availability": { 
+            "status": 1 
+            }, 
+            "permissions": { 
+            "resourceProvider": [ 
+                   { 
+                       "provider": "Microsoft.OperationalInsights/workspaces", 
+                       "permissionsDisplayText": "write permission is required.", 
+                       "providerDisplayName": "Workspace", 
+                       "scope": "Workspace", 
+                       "requiredPermissions": { 
+                       "write": true, 
+                       "delete": true 
+                    } 
+                } 
+            ],
+            "customs": [
+                {
+                    "name": "CylancePROTECT",
+                    "description": "must be configured to export logs via Syslog"
```

Comment:
Put a "." at the end. In the UX, it will show up as:

Workspace: write permission is required.
CylancePROTECT: must be configured to export logs via Syslog.

Reply:
typo corrected